app/src/main/java/org/akvo/flow/presentation/form/view/FormViewPresenter.kt

```diff
@@ -39,4 +41,21 @@ class FormViewPresenter @Inject constructor(): Presenter {
     ) {
         //TODO("not implemented")
     }
+
+    //language is saved per survey and not form???
```

Comment:
is this an open question?

Reply:
it's a fact, I will remove the question mark